### PR TITLE
Add events list

### LINKED ISSUE #126: Add events feed
To access https://github.com/sul-dlss/dor-services-app/blob/master/openapi.yml#L545-L546

As follow-on work, consider tackling https://github.com/sul-dlss/argo/issues/1773 (not in our zenhub board, but consider this in-scope and ready once this issue is closed).

### PR DESCRIPTION
**Pull request by jcoyne**
## Why was this change made?

Fixes #126

## Was the documentation (README, API, wiki, consul, etc.) updated?
yes

### CODE REVIEW

**Comment by jcoyne** [comment]
@ndushay how do you test a gem on stage?

**Comment by ndushay** [comment]
You test one of its consumers using the gem from this branch, exercising the changed methods, of course!

**Comment by jcoyne** [comment]
There are no consumers yet.

File: lib/dor/services/client/object.rb [hunk 1]
```diff
@@ -19,6 +19,10 @@ def metadata
           @metadata ||= Metadata.new(parent_params)
         end
 
+        def events
+          @events ||= Events.new(parent_params)
```
Comment (mjgiarlo) [inline lib/dor/services/client/object.rb]: Worth adding a related test here-ish? https://github.com/sul-dlss/dor-services-client/blob/master/spec/dor/services/client_spec.rb#L47-L75

**Pull request merged**